packages/@ember/-internals/glimmer/lib/syntax/outlet.ts

```diff
@@ -138,6 +222,7 @@ function stateFor(
     outlet: render.outlet,
     template,
     controller: render.controller,
+    model: render.model,
```

Comment:
Just noting, that changing this structure will likely require changes in `@ember/test-helpers`:

* https://github.com/emberjs/ember-test-helpers/blob/fdcf31ab04b908e4561eeea102fc07a6f483d640/addon-test-support/%40ember/test-helpers/setup-rendering-context.ts#L115-L141
* https://github.com/emberjs/ember-test-helpers/blob/fdcf31ab04b908e4561eeea102fc07a6f483d640/addon-test-support/ember-test-helpers/legacy-0-6-x/test-module-for-component.js#L227-L239

Reply:
model is allowed to be `undefined`, so it shouldn't cause any errors if it's not present